src/config/main.cfg

```diff
@@ -79,6 +79,9 @@ threads = 2
 [crypto_material]
 threads = 2
 
+[cve_lookup]
+threads = 3
```

Comment:
Were the changes in this file included by accident?

If not please put them in a seperate commit.

Reply:
That was not intentional -> removed